src/main/java/com/google/sps/data/AttendeeInterface.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+/** Interface for the Attendee class. */
+public interface AttendeeInterface {
+  String getSessionId();
+  String getScreenName();
+  Date getTimeLastPolled();
+  //Compares two objects to see if the values are the same.
```

Comment:
Space between // and first word.

Also this does not compare 2 objects. It compares an AttendeeInterface to itself